<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(58, 58, 58);
color: #FFFFFF;
font: 8pt;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="gammaNum">
     <property name="text">
      <string>1.0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSlider" name="wbSliderRed">
     <property name="minimum">
      <number>25</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="value">
      <number>130</number>
     </property>
     <property name="sliderPosition">
      <number>130</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="snComboBox"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="gainNum">
     <property name="text">
      <string>25</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="wbNumBlue">
     <property name="text">
      <string>2.8</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="gammaLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Gamma:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="snLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>S/N:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="customName">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Microscope</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="expLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Exposure:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QSlider" name="gammaSlider">
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>125</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="expNum">
     <property name="text">
      <string>16</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="gainLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Gain:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="wbNumRed">
     <property name="text">
      <string>1.2</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSlider" name="wbSliderBlue">
     <property name="minimum">
      <number>25</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="value">
      <number>280</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="autoLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSlider" name="gainSlider">
     <property name="maximum">
      <number>27</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="wbLabelRed">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>W/B Red:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="wbLabelBlue">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>W/B Blue:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="expSlider">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="expAuto">
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>♦</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="gainAuto">
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>♦</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="wbAuto">
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>♦</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="gammaAuto">
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>♦</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>expSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>expNum</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gainNum</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wbSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>wbNumRed</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gammaSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gammaNum</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
